uploadNeuroDB/NeuroDB/ImagingUpload.pm

```diff
@@ -478,7 +486,7 @@ Arguments:
 sub runCommand {
     my $this = shift;
     my ($command) = @_;
-    print "\n\n $command \n\n " if $this->{'verbose'};
+    print "\n\n $command \n\n " if $this->{verbose};
```

Comment:
On second thoughts, your object is also a hash array so I think the verbose property can be accessed both ways (with or without quotes). You sure it does not work *without* quotes (I think it should) ?